setup.py

```diff
@@ -84,9 +84,21 @@
 
 install_requires = [min_numpy_version]
 
-extras_require = {
-    'test': ['pytest-astropy']
-}
+extras_require = {}
+
+# All optional dependencies for the tests
+extras_require['test'] = ['pytest-astropy', 'pytest-xdist', 'pytest-mpl',
```

Comment:
We *could* keep the existing requirements files and instead do something like:
```suggestion
with open('pip-requrirements-test') as f:
    extras_require['test'] = [line for line in f.read().split('\n') if line.strip()!='']
```
?